packages/astrocite-bibtex/src/constants.ts

```diff
@@ -44,6 +44,8 @@ export const FIELD_MAP: ReadonlyMap<string, keyof Data> = new Map([
     ['series', 'collection-number'],
     ['title', 'title'],
     ['volume', 'volume'],
+    ['url', 'URL'],
+    ['doi', 'DOI'],
```

Comment:
It would be useful for someone to either look through the list of CSL fields and see which other fields might need to be added here (I can think of at least PMID and ISSN), or allow this Map to be extended by a parser option.

Reply:
I haven't looked at whether `parser.ts` handles any of these cases separately, but my own converter built on the astrocite parser also parses the following fields out of bibtex (and anything not on this list is still imported but without conversion rules applied):

```
abstract
address
annotation
archiveprefix
booktitle
chapter
comment
date
day
eprint
eprintclass
eprinttype
file
files
groups
holder
howpublished
institution
isbn
issn
issue
journal
journaltitle
keyword
keywords
langid
language
lastchecked
lista
location
month
nationality
notes
pages
pagetotal
primaryclass
review
school
series
shorttitle
slaccitation
subtitle
timestamp
titleaddon
type
urldate
year
```

Reply:
These fields all come from live samples gathered over the years.

Reply:
The list of csl variables is [here](https://docs.citationstyles.org/en/stable/specification.html#appendix-iv-variables) but I don't know anything about CSL (don't use it myself).

Reply:
Although one thing I do know about CSL (triggered by looking at the list I just pasted), if a language field of the bibtex reference imported is Engl-ish (american, australian, usenglish, etc), by spec the title-type fields of those entries would be Title Cased, where CSL expects title-type fields to be Sentence cased. Non-Engl-ish title fields are expected to be sentence cased. Wonderfully complicated.

Reply:
I use https://github.com/dsifford/csl-json/blob/master/index.d.ts as my source of truth of CSL fields. That might be useful for you to breeze over for anything else.

Otherwise, LGTM :+1:

Reply:
I'd prefer it if someone else would look over the mapping. I don't really know much about the semantics of the CSL fields.

Reply:
[These lines](https://github.com/dsifford/csl-json/blob/861dcbe574bf3f47cf85bb3413b711cd125d97a2/index.d.ts#L55-L136) are the ones that are relevant to look over.

@hubgit would you mind scanning these to see if there are other fields that we missed? After we get the okay, I'll go ahead and merge this.

Reply:
Merging so I don't forget about this. If you find something else, open a new PR. :+1: